doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
+Configuring a market requires three values beyond the base and quote type:
+
+| Parameter          | Meaning                  | Units                      |
+| ------------------ | ------------------------ | -------------------------- |
+| Lot size           | Order size granularity   | Subunits of base (`APT`)   |
+| Tick size          | Lot value granularity    | Subunits of quote (`USDC`) |
+| Minimum order size | Minimum limit order lots | Number of lots             |
+
+This section utilizes helpers from the Econia Python SDK.
+If you'd like to follow along, run in the nearest terminal:
+
+```bash
+pip3 install econia-sdk
+python3
+>>> from econia_sdk.utils.decimals import *
 ```
 
-Similarly, if a market has a decimal lot size of 0.0001 `APT`, than it can only support a decimal tick size down to 0.01 `USDC` per `APT`, because the increment in total quote amount for each additional lot or tick corresponds to a single `USDC` subunit:
+We'd like the value of one increment in order size to be worth about a penny, say \$0.00732.
+That means our lot size (in unit terms) should be 0.001 `APT`, which is worth said amount.
+To get the lot size in subunit terms so as to configure the market, we can use the SDK:
 
 ```python
->>> lot_size_decimal = dec('0.0001')
->>> tick_size_decimal = dec('0.01')
->>> quote_increment_decimal = lot_size_decimal * tick_size_decimal
->>> to_check = (quote_increment_decimal * 10 ** decimals_quote).normalize()
->>> to_check
-Decimal('1')
+>>> lot_size_unit = "0.001"
+>>> base_decimals = 8
+>>> lot_size = get_lot_size_integer(lot_size_unit, base_decimals)
+>>> lot_size
+100000
 ```
 
-Notably, this check amount is equivalent to the integer tick size:
+Now to configure the tick size; we know that this is the granularity in value of one lot, which is worth \$0.00732 per above.
+Therefore, we want a tick size of 0.00001 to achieve the lot value granularity required to express a price of \$0.00732 for one lot.
+Let's now get that in subunit terms using the SDK:
 
 ```python
->>> tick_size_integer = int(lot_size_decimal * tick_size_decimal
-...                         * 10 ** decimals_quote)
->>> tick_size_integer
-1
+>>> tick_size_unit = "0.00001"
+>>> quote_decimals = 6
+>>> get_tick_size_integer(tick_size_unit, quote_decimals)
+10
 ```
 
-A decimal tick size of 0.001 `USDC` per `APT` would not be supported, however, because at the lowest possible price of 1 tick, the increment in total quote amount for each additional lot could not be represented as an integer multiple of subunits:
+It's easy to tell if this lot size and tick size combination is reasonable by checking the minimum and maximum price under these conditions in nominal (unit) terms.
+Note that the minimum price is also equivalent to price granularity--this tells us the granularity of value for a whole unit of `APT` as opposed to just one lot (equal here to 100000 subunits of `APT`).
```

Comment:
`--` would ideally render as an m-dash but I'm guessing not here, and also maybe just change for more specificity:

"Note that the minimum price is also equivalent to price granularity, which tells us the granularity of value for a whole unit of APT as opposed to just one lot..."